tests/app/tests/test_webpack.py

```diff
@@ -210,6 +210,34 @@ def test_preload(self):
             '<script src="/static/django_webpack_loader_bundles/main.js" >'
             '</script>'), result.rendered_content)
 
+    def test_integrity(self):
+        self.compile_bundles('webpack.config.integrity.js')
+
+        loader = get_loader(DEFAULT_CONFIG)
+        with patch.dict(loader.config, {'INTEGRITY': True}):
```

Comment:
Please add another test that uses a `loader.config` without any `'INTEGRITY'` key. It should work as `'INTEGRITY': False`. This will ensure backwards compatibility.

Reply:
Added `test_integrity_missing_config`:

https://github.com/django-webpack/django-webpack-loader/pull/315/commits/e880336976ae77f750db2d38707c438411492e46#diff-1af3f1863bb5f5c3e50ccb014a6b83fd14ab4fe4dbc268c6285f119bbb9885b0R232-R254